src/Libraries/Microsoft.Extensions.AI.Abstractions/Embeddings/EmbeddingGenerationOptions.cs

```diff
@@ -1,11 +1,30 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Microsoft.Shared.Diagnostics;
+
 namespace Microsoft.Extensions.AI;
 
 /// <summary>Represents the options for an embedding generation request.</summary>
 public class EmbeddingGenerationOptions
 {
+    private int? _dimensions;
+
+    /// <summary>Gets or sets the number of dimensions requested in the embedding.</summary>
+    public int? Dimensions
```

Comment:
If a generator implementation doesn't support a particular option then configuring it is going to be a no-op, correct?

Reply:
Yes